08.Third-party-API-wrappers.md

```diff
@@ -5,8 +5,12 @@ You can also access the Gitter API through one of the 3rd party libraries that m
 ## .NET (C#)
 
 * [Gitter#](https://github.com/Odonno/gitter-api-pcl) - provides a Gitter API interface for use in Windows (Phone) apps, .NET Framework and Xamarin platforms
-	* [Gitter# Auth](https://github.com/Odonno/gitter-api-auth) - provides you a way to retrieve Token through OAuth2 process
+* [Gitter# Auth](https://github.com/Odonno/gitter-api-auth) - provides you a way to retrieve Token through OAuth2 process
 
 ## Java
 
 * [GitterRxJavaSDK](https://github.com/Gitteroid/GitterRxJavaSDK) - provides you a way to use Gitter API in Java with [RxJava](https://github.com/ReactiveX/RxJava) syntax
+
+## Go
+
+* [gitter](https://github.com/sromku/gitter) - Gitter API in Go. 
```

Comment:
It seems a bit odd to add a library named "gitter" for Gitter. For example, our node client for the API is [`node-gitter`](https://github.com/gitterHQ/node-gitter).

Some ideas (this is just for the name we would use in this list):

 - gitter for Go
 - gitter API wrapper
 - go-gitter